packages/admin_view/src/components/LexiconTable.tsx

```diff
@@ -0,0 +1,118 @@
+import { GetAllLexEntriesQuery, useGetAllLexEntriesQuery } from '../graphql/lexicon/lexicon.ts';
+import { DataGrid, GridColDef,DataGridProps } from '@mui/x-data-grid';
+import React, { useState } from 'react';
+import { Box, Stack, Button } from '@mui/material';
+import { alpha, createTheme, styled } from '@mui/material/styles';
+
+const theme = createTheme({
```

Comment:
Let's hold off on theme colors for now as it will be something we implement globally once we have a color palette from Ruby.